app/main/views/users.py

```diff
@@ -111,7 +148,7 @@ def create_user():
         db.session.rollback()
         abort(400, "Invalid supplier id")
 
-    return jsonify(users=user.serialize()), 200
+    return jsonify(users=user.serialize()), 201
```

Comment:
Why `201`? Nothing has been [created](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#201) here has it?